adapters/pulsepoint/pulsepoint.go

```diff
@@ -1,44 +1,196 @@
 package pulsepoint
 
 import (
-	"bytes"
-	"context"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"net/http"
 	"strconv"
-	"strings"
 
+	"bytes"
+	"context"
 	"github.com/mxmCherry/openrtb"
 	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/errortypes"
 	"github.com/prebid/prebid-server/openrtb_ext"
 	"github.com/prebid/prebid-server/pbs"
 	"golang.org/x/net/context/ctxhttp"
+	"io/ioutil"
+	"strings"
```

Comment:
Did you run `gofmt`? It should order the imports such that included packages (strings) are listed before external packages (github.com/mxmCherry/openrtb).

Reply:
done